src/panels/config/automation/trigger/types/ha-automation-trigger-webhook.ts

```diff
@@ -27,13 +86,51 @@ export class HaWebhookTrigger extends LitElement {
         name="webhook_id"
         .value=${webhookId}
         @value-changed=${this._valueChanged}
-      ></paper-input>
+      >
+        <ha-icon-button
+          @click=${this._copyUrl}
+          slot="suffix"
+          .label=${this.hass.localize(
+            "ui.panel.config.automation.editor.triggers.type.webhook.copy_url"
+          )}
+          .path=${mdiContentCopy}
+        ></ha-icon-button>
+      </paper-input>
+      <div class="helper-text">
```

Comment:
Use `ha-textfield` isntead of `paper-input` and you will be able to use the built-in helper text property. https://github.com/material-components/material-web/tree/master/packages/textfield#helper-text

Reply:
Thank you! That helped a lot.

![image](https://user-images.githubusercontent.com/289218/152937799-a420946f-f7b7-4027-8468-d03d179d065d.png)